doc-content/enterprise-only/getting-started-kogito/proc-kogito-microservices-deploying-on-ocp-console-file-build.adoc

```diff
@@ -0,0 +1,112 @@
+* You have access to the OpenShift web console with the necessary permissions to create and edit `KogitoBuild` and `KogitoRuntime`.
+
+.Procedure
+. Go to *Operators* -> *Installed Operators* and select *RHPAM Kogito Operator*.
+. To create the {KOGITO} build definition, on the operator page, select the *Kogito Build* tab and click *Create KogitoBuild*.
+. In the application window, use *Form View* or *YAML View* to configure the build definition.
++
+--
+At a minimum, define the application configurations shown in the following example YAML file:
+
+.Example YAML definition for a Quarkus application with {KOGITO} build
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rhpam.kiegroup.org/v1 # {KOGITO} API for this service
+kind: KogitoBuild  # Application type
+metadata:
+  name: example-quarkus  # Application name
+spec:
+  type: LocalSource
+----
+
+.Example YAML definition for a Spring Boot application with {KOGITO} build
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rhpam.kiegroup.org/v1 # {KOGITO} API for this service
+kind: KogitoBuild  # Application type
+metadata:
+  name: example-springboot  # Application name
+spec:
+  runtime: springboot
+  type: LocalSource
+----
+--
+
+. After you define your application data, click *Create* to generate the {KOGITO} build.
++
+Your application is listed in the *{KOGITO}Builds* page. You can select the application name to view or modify application settings and YAML details.
+
+. Upload the file asset using the following command:
++
+--
+[source]
+----
+$ oc start-build example-quarkus-builder --from-file=<file-asset-path> -n namespace
+----
+
+* `file-asset-path` is the path of the file asset that you want to upload.
+* `namespace` is the namespace where `KogitoBuild` is created.
+--
+
+. To create the {KOGITO} microservice definition, on the operator page, select the *Kogito Runtime* tab and click *Create KogitoRuntime*.
+. In the application window, use *Form View* or *YAML View* to configure the microservice definition.
++
+--
+At a minimum, define the application configurations shown in the following example YAML file:
+
+.Example YAML definition for a Quarkus application with {KOGITO} microservices
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rhpam.kiegroup.org/v1 # {KOGITO} API for this microservice
+kind: KogitoRuntime  # Application type
+metadata:
+  name: example-quarkus  # Application name
+----
+
+.Example YAML definition for a Spring Boot application with {KOGITO} microservices
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rhpam.kiegroup.org/v1 # {KOGITO} API for this microservice
+kind: KogitoRuntime  # Application type
+metadata:
+  name: example-springboot  # Application name
+spec:
+  runtime: springboot
+----
+--
+
```

Comment:
I would add this NOTE here again: https://github.com/hmanwani-rh/kie-docs/pull/60/files#diff-4068d8298fe916b95c6fd155776d7e23c666635c961e6533dd79ffae97b17b5aR111-R114

Reply:
Done, added the following note in file build section:

[NOTE]
====
In this case, the application is built from a file and deployed using KogitoRuntime. You must ensure that the application name is same in `KogitoBuild` and `KogitoRuntime`.
====